index.js

```diff
@@ -223,35 +223,39 @@ function typeHasher(options, writeTo, context){
           throw new Error('Unknown object type "' + objType + '"');
         }
       }else{
-        var keys = Object.keys(object);
-        if (options.unorderedObjects) {
-          keys = keys.sort();
-        }
-        // Make sure to incorporate special properties, so
-        // Types with different prototypes will produce
-        // a different hash and objects derived from
-        // different functions (`new Foo`, `new Bar`) will
-        // produce different hashes.
-        // We never do this for native functions since some
-        // seem to break because of that.
-        if (options.respectType !== false && !isNativeFunction(object)) {
-          keys.splice(0, 0, 'prototype', '__proto__', 'constructor');
-        }
-
-        if (options.excludeKeys) {
-          keys = keys.filter(function(key) { return !options.excludeKeys(key); });
-        }
+        if (objType === 'file') {
+          write('file:' + object.name + "," + object.size + "," + object.type + "," + object.lastModfied);
```

Comment:
I think hashing the content would be the most important part here

Reply:
I agree. How to access file's content?

Reply:
Hm, I’m not sure that that’s possible to do synchronously (file.arrayBuffer() is `async`) – so probably this is the best we can do